src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+
+const GET_ROCKETS = 'spaceX/rockets/GET_ROCKETS';
+
+const initialRockets = [];
+
+export const getRockets = () => async (dispatch) => {
+  const res = await axios.get('https://api.spacexdata.com/v3/rockets');
+  const data = await res.data;
+  if (data) {
+    const rockets = data.map((rocket) => ({
+      id: rocket.rocket_id,
+      name: rocket.rocket_name,
+      description: rocket.description,
+      flickrImage: rocket.flickr_images[0],
+
+    }));
+    dispatch({
+      type: GET_ROCKETS,
+      rockets,
+    });
+  }
+};
+
+const rocketsReducer = (state = initialRockets, action) => {
+  switch (action.type) {
+    case GET_ROCKETS:
+      return [...action.rockets];
```

Comment:
Here you're clearing the rockers in store whenever you fetch rockets again. 
Please, separate clearing the store and adding rockets into different actions, that way you can choose to fetch rockets without discarding your current data (that may have meaningful changes during the user's interaction with the page).